### PR TITLE
[TR] Add osu!supporter

### PR DESCRIPTION
**Pull request by 3dsosu**
Turkish translation for osu!supporter wiki page. It didn't look like it was going to get updated any time soon, so I wanted to push the translation.

### CODE REVIEW

File: wiki/osu!supporter/tr.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+### Süreyi seçmek
+
+![osu!supporter satın almak](img/selecting-duration.jpg "osu!supporter için kullanıcıyı ve süreyi seçmek")
+
+Kaydırıcının üzerinde bulunan kullanıcı kartı osu!supporter'ı aldığınız kullanıcıyı gösterir. Varsayılan olarak, bu sizin kullanıcı kartınız olmalı. Bir arkadaşınıza hediye etmek için, arkadaşınızın kullanıcı ismini (ID'sini değil) kullanıcı kartının altındaki gri kutuya girin. Bu yapıldıktan sonra, kullanıcı kartı girilen kullanıcı isminin hesabıyla değiştirilecektir. Buradan sonra, hediye etmek istediğiniz osu!supporter süresini seçebilirsiniz. Ancak, birden fazla kişiye yapılan hediyelerde bir promosyon yapılmamaktadır.
```
Comment (zeusminus) [inline wiki/osu!supporter/tr.md:27]:
```suggestion
Kaydırıcının üzerinde bulunan kullanıcı kartı osu!supporter aldığınız kullanıcıyı gösterir. Varsayılan olarak, bu sizin kullanıcı kartınız olmalı. Bir arkadaşınıza hediye etmek için, arkadaşınızın kullanıcı ismini (kullanıcı ID'sini değil) kullanıcı kartının altındaki gri kutuya girin. Bu yapıldıktan sonra, kullanıcı kartı girilen kullanıcı isminin hesabıyla değiştirilecektir. Buradan sonra, hediye etmek istediğiniz osu!supporter süresini seçebilirsiniz. Ancak, birden fazla kişiye gönderilen hediyelerde bir promosyon yapılmamaktadır.
```

File: wiki/osu!supporter/tr.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+osu!supporter'ı bu ekrandan sadece bir kişiye hediye edebilirsiniz. Başka bir kullanıcıya hediye etmek için, [osu!supporter mağaza sayfası](https://osu.ppy.sh/store/products/supporter-tag)'na geri gidip başka bir kullanıcıyı ekleme adımlarını tekrar yapmalısınız.
```
Comment (zeusminus) [inline wiki/osu!supporter/tr.md:29]:
```suggestion
Bu ekrandan tek seferde sadece bir kişiye osu!supporter hediye edebilirsiniz. Başka bir kullanıcıya hediye etmek için, [osu!supporter mağaza sayfasına](https://osu.ppy.sh/store/products/supporter-tag) geri gidip başka bir kullanıcıyı eklemek için gereken adımları tekrarlamalısınız.
```

File: wiki/osu!supporter/tr.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+### Ödeme yapmak
+
+İşiniz bittiği zaman, `Sepete ekle` tuşuna basarak osu!store mağaza sepetinize ekleyin.
+
+![osu!store mağaza sepeti](img/shopping-cart.jpg "içinde flyte için osu!supporter bulunan bir osu!store mağaza sepeti")
+
+Satın alımı yapmak için hazır olduğunuz zaman, [mağaza sepeti](https://osu.ppy.sh/store/cart)'nize gidin ve `Ödeme` tuşuna basın. Ardından adımları takip ederek ödemenizi gerçekleştirin.
```
Comment (zeusminus) [inline wiki/osu!supporter/tr.md:37]:
```suggestion
Satın alımı yapmak için hazır olduğunuz zaman, [mağaza sepetinize](https://osu.ppy.sh/store/cart) gidin ve `Ödeme` tuşuna basın. Ardından adımları takip ederek ödemenizi gerçekleştirin.
```

File: wiki/osu!supporter/tr.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+![osu!supporter satın almak](img/selecting-duration.jpg "osu!supporter için kullanıcıyı ve süreyi seçmek")
+
+Kaydırıcının üzerinde bulunan kullanıcı kartı osu!supporter'ı aldığınız kullanıcıyı gösterir. Varsayılan olarak, bu sizin kullanıcı kartınız olmalı. Bir arkadaşınıza hediye etmek için, arkadaşınızın kullanıcı ismini (ID'sini değil) kullanıcı kartının altındaki gri kutuya girin. Bu yapıldıktan sonra, kullanıcı kartı girilen kullanıcı isminin hesabıyla değiştirilecektir. Buradan sonra, hediye etmek istediğiniz osu!supporter süresini seçebilirsiniz. Ancak, birden fazla kişiye yapılan hediyelerde bir promosyon yapılmamaktadır.
+
+osu!supporter'ı bu ekrandan sadece bir kişiye hediye edebilirsiniz. Başka bir kullanıcıya hediye etmek için, [osu!supporter mağaza sayfası](https://osu.ppy.sh/store/products/supporter-tag)'na geri gidip başka bir kullanıcıyı ekleme adımlarını tekrar yapmalısınız.
+
+### Ödeme yapmak
+
+İşiniz bittiği zaman, `Sepete ekle` tuşuna basarak osu!store mağaza sepetinize ekleyin.
```
Comment (zeusminus) [inline wiki/osu!supporter/tr.md:33]:
```suggestion
İşiniz bittiği zaman, `Sepete Ekle` tuşuna basarak osu!store mağaza sepetinize ekleyin.
```

File: wiki/osu!supporter/tr.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+- `{kullanıcı adı} bir osu!supporter oldu - cömertliğiniz için teşekkürler!` eğer o hesap ilk kez osu!supporter olduysa.
+- `{kullanıcı adı} tekrardan osu!'yu desteklemeyi seçti! - cömertliğiniz için teşekkürler!` eğer hesap daha önceden osu!supporter olduysa.
+
+### Kullanıcı sayfası ve kullanıcı kartı
+
+![osu!supporter'lı kullanıcı sayfası](img/userpage.jpg "osu!supporter'lı kullanıcı sayfası")
+
+![osu!supporter'lı kullanıcı kartı](img/usercard.png "osu!supporter'lı kullanıcı kartı")
+
+osu!web'de, osu!supporter rozeti avatarınızın yanında ve kullanıcı kartınızda ülkenizin bayrağının yanında yer alır.
```
Comment (zeusminus) [inline wiki/osu!supporter/tr.md:52]:
```suggestion
osu!web'te, osu!supporter rozeti avatarınızın yanında ve kullanıcı kartınızda ülkenizin bayrağının yanında yer alır.
```

File: wiki/osu!supporter/tr.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+### Supporter durumunuzu kontrol etmek
```
Comment (zeusminus) [inline wiki/osu!supporter/tr.md:54]:
```suggestion
### Destek durumunuzu kontrol etmek
```

File: wiki/osu!supporter/tr.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+![Üç ay süre kalan osu!spporter tagı](img/status.jpg "Üç ay süre kalan osu!spporter tagı")
```
Comment (zeusminus) [inline wiki/osu!supporter/tr.md:56]:
```suggestion
![Üç ay süre kalan osu!supporter etiketi](img/status.jpg "Üç ay süre kalan osu!supporter etiketi")
```

File: wiki/osu!supporter/tr.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+*Not: Bu bölüm sadece osu!supporter elde ettiğinizde gözükür.*
+
+Geriye kalan osu!supporter durumunuza, toplam katkınıza ve tag satın alımlarınıza bakmak için, [oyunu destekle](https://osu.ppy.sh/home/support) sayfasına gidin ve aşağı inerek yukarıda ki resimdekine benzer bir bölüm göreceksiniz.
```
Comment (zeusminus) [inline wiki/osu!supporter/tr.md:60]:
```suggestion
Geriye kalan osu!supporter durumunuza, toplam katkınıza ve etiket satın alımlarınıza bakmak için, [oyunu destekle](https://osu.ppy.sh/home/support) sayfasına gidin ve aşağı inerek yukarıdaki resimdekine benzer bir bölüm göreceksiniz.
```

**Comment by 3dsosu** [comment]
Since I've made the last update on the document a bit harder to understand (I'm sorry). I might as well just comment on what changes I've done to it. (Sorry, it's a bit confusing for me)

I've basically fixed these following lines:

"Satın alımı yapmak için hazır olduğunuz zaman, [mağaza sepeti**nize**](https://osu.ppy.sh/store/cart) gidin ve `Ödeme` tuşuna basın. Ardından adımları takip ederek ödemenizi gerçekleştirin."

File: wiki/osu!supporter/tr.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+Bu ekrandan tek seferde sadece bir kişiye osu!supporter hediye edebilirsiniz. Başka bir kullanıcıya hediye etmek için, [osu!supporter mağaza sayfasına](https://osu.ppy.sh/store/products/supporter-tag) geri gidip başka bir kullanıcıyı eklemek için gereken adımları tekrarlamalısınız.
+
+### Ödeme yapmak
+
+İşiniz bittiği zaman, `Sepete Ekle` tuşuna basarak osu!store mağaza sepetinize ekleyin.
+
+![osu!store mağaza sepeti](img/shopping-cart.jpg "içinde flyte için osu!supporter bulunan bir osu!store mağaza sepeti")
+
+Satın alımı yapmak için hazır olduğunuz zaman, [mağaza sepetinize](https://osu.ppy.sh/store/cart) ve `Ödeme` tuşuna basın. Ardından adımları takip ederek ödemenizi gerçekleştirin.
```
Comment (zeusminus) [inline wiki/osu!supporter/tr.md:37]:
```suggestion
Satın alımı yapmak için hazır olduğunuz zaman, [mağaza sepetinize](https://osu.ppy.sh/store/cart) gidin ve `Ödeme` tuşuna basın. Ardından adımları takip ederek ödemenizi gerçekleştirin.
```
seems like this was deleted by accident while I was adding my suggestions, my bad

**Pull request merged**